internal/configuration/reader.go

```diff
@@ -67,3 +80,22 @@ func Read(configPath string) (*schema.Configuration, []error) {
 
 	return &configuration, nil
 }
+
+func generateConfigFromTemplate(configPath string) error {
+	f, err := cfg.Open("config.template.yml")
+	if err != nil {
+		return fmt.Errorf("Unable to open config.template.yml: %v", err)
+	}
+
+	b, err := ioutil.ReadAll(f)
+	if err != nil {
+		return fmt.Errorf("Unable to read config.template.yml: %v", err)
+	}
+
+	err = ioutil.WriteFile(configPath, b, 0644) //nolint:gosec // Configuration file does not need to be 0600.
```

Comment:
What would it cost to be more restrictive by default using 0600? I mean in a good setup admins should use secrets stored in files as suggested in the doc but what if the admin still provide passwords in the config? They will be available to all users on the host. I think it's safer to be on the restrictive side.
Plus, I don't foresee any direct side effect due to using 0600 instead of 0644. What do you think?

Reply:
There's no particularly downside to doing this so I've adjusted accordingly.